src/index.tsx

```diff
@@ -1,14 +1,11 @@
-import { Redis } from "@upstash/redis/cloudflare";
 import { Hono } from "hono";
-import { env } from "hono/adapter";
-
-type Env = {
-	UPSTASH_REDIS_REST_URL: string;
-	UPSTASH_REDIS_REST_TOKEN: string;
-};
+import { api } from "./api";
+import { redis } from "./middleware";
 
 const app = new Hono();
 
+app.use("/api/*", redis());
```

Comment:
will `/api/*` only be for redis? maybe:
```suggestion
app.use("/api/kv/", redis());
```

Reply:
I'm not a huge fan of making the api routes specific to the database we use